packages/contracts-bedrock/snapshots/storageLayout/RISCV.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "bytes": "20",
+    "label": "oracle",
+    "offset": 0,
+    "slot": "0",
+    "type": "contract IPreimageOracle"
```

Comment:
Question, unrelated to PR so not a blocker: In MIPS the preimage oracle is an immutable, here's it's in storage. Any reason for that difference?